metatron-app/metatron-discovery
# #3295 support responsive web of external login page

### Description

Support external login page for responsive web

**Related Issue** : #32955


### How Has This Been Tested?

1. Register the oauth client. (can register the logo, favicon, background image, copyright html)
2. Login the the oauth client.
3. Check the login page at many resolutions.
4. Reset password and login the client. (Check the change password popup).
5. Login the oauth client and login the same oauth client at the different IP. (Check the notification popup)

- Over width 500px
![metatron_Discovery](https://user-images.githubusercontent.com/45194478/88353143-551de600-cd97-11ea-82c5-df599d703108.jpg)

- Under width 500px
![metatron_Discovery](https://user-images.githubusercontent.com/45194478/88353461-7df2ab00-cd98-11ea-8f59-f324b2e3b6f7.jpg)

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
